sv.c

```diff
@@ -4275,24 +4277,17 @@ Perl_sv_setsv_flags(pTHX_ SV *dsv, SV* ssv, const I32 flags)
         SvREFCNT_dec(old_rv);
         return;
     }
-
+/*
 #if NVSIZE <= IVSIZE
     both_type = (stype | dtype);
 #endif
-
+*/
     if (UNLIKELY(both_type == SVTYPEMASK)) {
-        if (SvIS_FREED(dsv)) {
-            Perl_croak(aTHX_ "panic: attempt to copy value %" SVf
-                       " to a freed scalar %p", SVfARG(ssv), (void *)dsv);
-        }
-        if (SvIS_FREED(ssv)) {
-            Perl_croak(aTHX_ "panic: attempt to copy freed scalar %p to %p",
-                       (void*)ssv, (void*)dsv);
-        }
+        S_sv_setsv_flags_croak(dsv, ssv);
```

Comment:
This and the other calls below should be sv_setsv_flags_croak() not S_sv_setsv_flags_croak() for threaded builds to build.

Reply:
Thanks, tidying up now.